scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -201,14 +201,21 @@ private int startTasks(Policy currentPolicy, Map<JobId, JobDescriptor> jobMap, M
             //get rmState and update it in scheduling policy
             Set<String> freeResources = getFreeResources(currentPolicy);
             //if there is no free resources, stop it right now without starting any task
-            if (freeResources.isEmpty()) {
-                return 0;
-            }
```

Comment:
Why did you remove this condition ? 
(you forgot to remove the comment)

Reply:
Thanks. Indeed i forgot to remote a comment. But condition is below now.